assets/shop/js/_visa_mobile.js

```diff
@@ -10,6 +13,18 @@ document.addEventListener('DOMContentLoaded', () => {
     return;
   }
 
+  phoneNumber.addEventListener('keypress', function(e) {
+    if (!/[0-9]/.test(e.key)) {
+      e.preventDefault();
+    }
+  });
```

Comment:
Prevents typing a-z

Reply:
I think same could be done with html attribute type="number" but it might be dependent on if we want + sign into input etc. So I am not forcing this change just putting it as side note. Also there is attribute pattern="some regex" which will validate input on submit :)